### PR TITLE
[i18n] Arabic: fix spelling / Uzbek: add reciters' names

### PR DESCRIPTION
**Pull request by ozbek**

### CODE REVIEW

File: app/src/main/res/values-ar/readers.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <string name="qari_hani_rifai">هاني الرفاعي</string>
     <string name="qari_husary_mujawwad">الحصري مجود</string>
     <string name="qari_hudhayfi">الحذيفي</string>
-    <string name="qari_muaiqly">ماهر المعيلقي</string>
+    <string name="qari_muaiqly">ماهر_المعيقلي</string>
```
Comment (ahmedre) [inline app/src/main/res/values-ar/readers.xml]: why did you add an `_` here?

Comment (ozbek) [inline app/src/main/res/values-ar/readers.xml]: Sorry, underscore was unintentional. I only wanted to correct the sequence
of letters.

Comment (ahmedre) [inline app/src/main/res/values-ar/readers.xml]: excellent catch, may Allah reward you greatly!
